### PR TITLE
Added txt as md plugin

### PR DESCRIPTION
**Pull request by deathau**

### CODE REVIEW

File: community-plugins.json [hunk 1]
```diff
@@ -809,5 +809,13 @@
         "description": "This plugin enables you to complement input in markdown files",
         "repo": "tadashi-aikawa/obsidian-various-complements-plugin",
         "branch": "main"
-    }
+    },
+    {
+        "id": "txt-as-md-obsidian",
+        "name": "txt as md",
+        "author": "death_au",
+        "description": "Edit ini files in Obsidian",
```
Comment (liamcain) [inline community-plugins.json:817]: Wrong description 😄

Comment (deathau) [reply community-plugins.json]: Whoops. Thanks for the catch

File: community-plugins.json [hunk 2]
```diff
@@ -809,5 +809,13 @@
...
-    }
+    },
+    {
+        "id": "txt-as-md-obsidian",
+        "name": "txt as md",
+        "author": "death_au",
+        "description": "Edit txt files as markdown in Obsidian",
+        "repo": "deathau/txt-as-md-obsidian",
+        "branch": "main"
+    },
```
Comment (lishid) [inline community-plugins.json:820]: Might need to remove this extra comma

**Review by lishid — CHANGES_REQUESTED** [review]

File: community-plugins.json (hunk 2 above)
Comment (deathau) [reply community-plugins.json]: done

**Pull request merged**